scss/bootstrap/components/_editable-content.scss

```diff
@@ -0,0 +1,31 @@
+.crm-editable-enabled:not(.crm-editable-editing):hover {  
```

Comment:
@AkA84 , should we keep pencil edit on data-type boolean or select,
When an editable content gets converted to dropdown box?

![test-1-hw-373](https://user-images.githubusercontent.com/26058635/29550728-e91b0fd6-872c-11e7-8132-4a6c2a36b94d.png)
Transforms to,
![test-2-hw-373](https://user-images.githubusercontent.com/26058635/29550730-f0448102-872c-11e7-9b9c-e0bd53f841e4.png)

I have checked other changes and fixed it. Let me know before we commit to this.